requirements.txt

```diff
@@ -4,14 +4,11 @@ dipy
 # https://github.com/tensorflow/tensorflow/issues/44467
 h5py~=2.10.0
 Keras==2.3.1
-ivadomed
+git+https://github.com/ivadomed/ivadomed.git@jn/1042-loosen-requirements-versions
```

Comment:
I temporarily pointed SCT to my test branch on ivadomed. It looks like the installation went OK! 

![Screenshot from 2022-01-18 11-57-54](https://user-images.githubusercontent.com/16181459/149983084-3a7c1ac3-c848-4bb1-967a-8f3eecae72fd.png)

The reason for the test failures is not a real issue: It's just because `sct_check_dependencies` isn't expecting an installation from `git`. :P

Reply:
I've temporarily addressed this in f7207b2, but it will be undone before this PR is merged.